packages/current-branch-name/package.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "current-branch-name",
+  "name": "@octopusdepoy/current-branch-name",
```

This looks like it should probably be

```suggestion
  "name": "@octopusdeploy/current-branch-name",
```